web/tests/compose_ui.test.js

```diff
@@ -434,6 +435,27 @@ run_test("quote_message", ({override, override_rewire}) => {
 
     reset_test_state();
 
+    // If forwarding a message, the quoted message should be inserted into
+    // an empty compose box, even if compose box wasn't previously empty.
+    let new_message = false;
+    override_rewire(compose_actions, "start", (opts) => {
```

Comment:
Does this need `rewire`?

Reply:
Changing to just override() yields this: 
![image](https://github.com/user-attachments/assets/91d6872f-def0-4b05-8391-e9e7335f8fe7)

Reply:
Probably not without more refactoring. We're just aiming to avoid `rewire` unless it's actually required; future test infrastructure goals will require replacing it.